app/assets/stylesheets/admin/all.scss

```diff
@@ -5,7 +5,6 @@
  *
 
  *= require normalize
- *= require skeleton
```

Comment:
this can be removed from vendor as well?

Reply:
This is actually in `spree_core`. Maybe I need to rebase and have another look...

Reply:
Done.

Reply:
yes, I brought it from spree recently :+1: